pennylane/beta/plugins/tf_ops.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Utility functions and numerical implementations of quantum operations TensorFlow devices.
+"""

```

Comment:
@co9olguy: this was copied from your open PR #599, since it will be useful for both this device and `default.tensor.tf`.